events.lua

```diff
@@ -1,6 +1,14 @@
 function register_events(event)
 	gauge_tick:set(game.tick)
 
+        for _, surface in pairs(game.surfaces) do
```

Comment:
i think seed can't change during a live game, so it's best to just fill this value at save loading instead of query it for each n tick

Reply:
like [75cf057](https://github.com/remijouannet/graftorio2/pull/24/commits/75cf057c1ff4f9b92e5de3fba7fd9af093e5e77f)?